jobs/JGFS_TC_GENESIS

```diff
@@ -34,14 +34,14 @@ export pgmerr=errfile
 # SENDCOM  - Copy Files From TMPDIR to $COMOUT
 # SENDDBN  - Issue DBNet Client Calls
 ####################################
-export SENDECF=${SENDECF:-YES}
+export SENDECF=${SENDECF:-NO}
 export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
+export SENDDBN=${SENDDBN:-NO}
 
 ####################################
 # Specify Execution Areas
 ####################################
-export HOMEens_tracker=${HOMEens_tracker:-${NWROOT:?}/ens_tracker.${ens_tracker_ver}}
+export HOMEens_tracker=${HOMEens_tracker:-${NWROOT:?}}
```

Comment:
Check instances of `HOMEens_tracker` being set. You should set it like this:
```
export HOMEens_tracker=${HOMEens_tracker:${PACKAGEROOT}/ens_tracker.${ens_tracker_ver}}
```
Example from the GFS:
```
export HOMEgfs=${HOMEgfs:-${PACKAGEROOT}/gfs.${gfs_ver}}
```
The `NWROOT` variable has been replaced with `PACKAGEROOT` in operations on WCOSS2. It's defined as `/lfs/h1/ops/prod/packages` by operations.